### PR TITLE
Completed sql-join exercise. -Paul Glorioso

### PR DESCRIPTION
**Pull request by PaulGlorioso**
Screenshot 2022-01-27 150004
where-are-they
Screenshot 2022-01-27 154259
Screenshot 2022-01-27 160351
Screenshot 2022-01-27 161905

### CODE REVIEW

File: sql-join/magic-mallrats.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+select "firstName",
+       "lastName"
+  from "customers"
+    join "rentals"
+      on "customers"."customerId" = "rentals"."customerId"
+    join "inventory"
+      on "inventory"."inventoryId" = "rentals"."inventoryId"
+    join "films"
+      on "films"."filmId" = "inventory". "filmId"
```
Comment (scbowler) [inline sql-join/magic-mallrats.sql]: It is cleaner and easier to use the `using` syntax.

```SQL
  from "customers"
    join "rentals" using ("customerId")
    join "inventory" using ("inventoryId")
    join "films" using ("filmId")
 ```

File: sql-join/sassy-cast.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+select "firstName",
+       "lastName"
+  from "actors"
+    join "castMembers"
+      on "actors"."actorId" = "castMembers"."actorId"
+    join "films"
+      on "films"."filmId" = "castMembers"."filmId"
```
Comment (scbowler) [inline sql-join/sassy-cast.sql]: Same here

File: sql-join/what-and-when.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+select "films"."releaseYear",
+       "genres"."name"
+  from "films"
+    join "filmGenre"
+      on "films"."filmId" = "filmGenre"."filmId"
+    join "genres"
+      on "genres"."genreId" = "filmGenre"."genreId"
```
Comment (scbowler) [inline sql-join/what-and-when.sql]: Same here

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just a few updates, it is better to use the "using" syntax for joins. Let me know if you have any questions.

**Comment by PaulGlorioso** [comment]
Screenshot 2022-01-27 164104

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**